stack/eslint-config/patch-eslint-resolver.js

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
```

Comment:
Can this be written in TypeScript?

Reply:
Historically there were a couple reasons why I didn't do this:

- It's a monkey patch for internal code, so there aren't going to be types for most of the data structures
- `eslint-config` is a Rush cyclic dependency with weird peer dependency relationships on the `typescript` engine, so I had trouble getting RSC to work correctly

That said, I recently revisited the original GitHub issue where I proposed this fix, and apparently a number of people are using the patch in production. Thus I'm thinking we should move it into its own package (`@rushstack/eslint-patch`?) so that people can reuse it separately from `@rushstack/eslint-config` (which has a bunch of NPM dependencies).

Reply:
Fixed as of 13a08eaf3af9783387ffc982ba6dcb7b97cc1dd9